src/Build/BackEnd/BuildManager/BuildRequestData.cs

```diff
@@ -15,6 +15,7 @@ namespace Microsoft.Build.Execution
 {
     /// <summary>
     /// Flags providing additional control over the build request
+    /// This enum has a copt on Microsoft.Build.Framework. Any modifications must be mirrored on the copy.
```

Comment:
Could we [type forward](https://learn.microsoft.com/dotnet/standard/assembly/type-forwarding) instead of having two copies of this?

Reply:
Nice trick. I wasn't aware - so I advised copying.

To specify more - we'll need to move the type to Microsoft.Build.Framework assembly (keep the namespace and naming) and use the `TypeForwardedToAttribute` in the Microsoft.Build assembly. (This is because the Microsoft.Build.Framework can be used standalone - so it needs to own the type)